src/StatisticsAnalysisTool/ao-pre-release-update-check.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should not change this file, please undo these changes.

It's best to always use the latest dev branch and then make your changes.